deployment/terraform/modules/docker_host/main.tf

```diff
@@ -33,6 +31,13 @@ resource "aws_instance" "docker_host" {
   root_block_device {
     volume_size = "20"
   }
+  tags                   = "${merge(
+    var.tags, map("Name", "${terraform.workspace} docker host")
+  )}"
+  volume_tags            = "${merge(
+    var.tags, map("Name", "${terraform.workspace} docker host")
+  )}"
```

Comment:
Sorry for just not getting the syntax, but `tags` had been a map type, and is it now a string. Or after interpolation is it transformed into a map?

Reply:
Would it be worth defining a variable rather than repeating these lines?

Reply:
Interpolation works fine:
```hcl
$ terraform console
> map("a", "b")
{
  "a" = "b"
}
> "${map("a", "b")}"
{
  "a" = "b"
}
```
This expression is used only here. Declaring it as a variable would require putting it in a  separate `local` block outside of this resource which would reduce readability.